packages/backend/src/misc/is-instance-muted.ts

```diff
@@ -0,0 +1,7 @@
+export function isInstanceMuted(note: any, muted_instances: Set<string>): boolean {
+	if (muted_instances.has(note?.user?.host ?? '')) return true;
+	if (muted_instances.has(note?.reply?.user?.host ?? '')) return true;
+	if (muted_instances.has(note?.renote?.user?.host ?? '')) return true;
+
+	return false;
+}
```

Comment:
```suggestion
import { Packed } from "./schema";

export function isInstanceMuted(note: Packed<'Note'>, mutedInstances: Set<string>): boolean {
	if (mutedInstances.has(note?.user?.host ?? '')) return true;
	if (mutedInstances.has(note?.reply?.user?.host ?? '')) return true;
	if (mutedInstances.has(note?.renote?.user?.host ?? '')) return true;

	return false;
}
```

- Add type definition (https://github.com/misskey-dev/misskey/pull/7712/files/9eed7816169f46babb34c69145a8c61e9b371e7f#r702389375)
- Use uppercase